test/unit/browser/disposable_test.dart

```diff
@@ -350,5 +352,69 @@ void main() {
           doesCallbackReturn: false);
       controller.close();
     });
+
+    group('Event subscription helper methods', () {
+      test(
+          'subscribeToDocumentEvent should remove same listener when thing is disposed',
+          () async {
+        var document = new MockEventTarget();
+        var event = 'event';
+        EventListener callback = (_) {};
+        var useCapture = true;
+
+        thing.subscribeToDocumentEvent(event, callback,
+            documentObject: document, useCapture: useCapture);
+        verify(document.addEventListener(event, callback, useCapture));
+        await thing.dispose();
+        verify(document.removeEventListener(event, callback, useCapture));
+      });
+
+      test(
+          'subscribeToDomElementEvent should remove listener when thing is disposed',
+          () async {
+        var element = new Element.div();
+        int numberOfEventCallbacks = 0;
+        EventListener callback = (_) {
+          numberOfEventCallbacks++;
+        };
+        var event = new Event('event');
```

Comment:
It might be nice to extract the string "event" into a variable like `eventName` to be consistent with the previous test and the event objects in this test. #nit